router-bridge/src/error.rs

```diff
@@ -0,0 +1,18 @@
+/*!
+# Errors raised by the `router-bridge` when trying to run `javascript`.
+*/
+
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+#[derive(Error, Serialize, Deserialize, Debug, Clone)]
+/// An error which occurred within the bridge.
+///
+/// This does not include JS domain related errors, such as [`GraphQLError`].
+pub enum Error {
```

Comment:
I feel like we should be able to put together a better name for this. Is it `DenoRuntimeError`? Also, is it really an enum or is there only ever going to be one thing? In which case:
`pub struct DenoRuntimeError(String);`

Reply:
Oh that's interesting, I think we might reuse it for the with_parameter, lemme check